### ISSUE TITLE
Bug: Transparent Window Border

### ISSUE THREAD

**Issue by MajorXAML** [Kinnara/ModernWpf#441]
Hi all, 

I found a small inaccuracy, the thing is that the Window border does not support transparency. According to win ui 2.5 and 3.0 the window border has a certain level of transparency (they don't use solid color).  I tried to fix it but unfortunately `AllowsTransparency="True"` doesn't work without `WindowStyle="None"` and `WindowStyle="None"` causes bug when maximizing the window (there is 1 pixel of solid color around the entire window)

tl:dr maybe someone knows how to make a transparent border work in a [Window](https://github.com/Kinnara/ModernWpf/blob/master/ModernWpf/Styles/Window.xaml)?

**Comment by MajorXAML** [comment]
example (windows 10)
![image](https://user-images.githubusercontent.com/93913816/159135590-d4019819-0d1e-4d8c-a907-7a22a54cf32c.png)

**Comment by Kinnara** [comment]
Confirmed against current `master` / the active 1.x window contract.

There are two separate behaviors here:

* The reported one-pixel client border while maximized is no longer present in the current template: the `WindowState=Maximized` trigger sets `LayoutRoot.BorderThickness=0`. The newer captionless path also collapses `TitleBarControl`, detaches `WindowChrome`, disables `WindowHelper.FixMaximizedWindow`, and places content at `(0,0)`.
* Making a WPF client border reveal the desktop behind the HWND is not a brush-only change. WPF requires `AllowsTransparency=true`, which in turn requires `WindowStyle=None` and a layered/captionless window. That conflicts with ModernWpf's supported 1.x shell contract, which retains `WindowChrome`, native resizing/snap behavior, High Contrast behavior, and the supported `net462`/Windows 7 path. The active source authority for stock WPF `Window` is official WPF Fluent, not the old UWP/WinUI system window frame.

Validation on `net8.0-windows7.0` Release:

`dotnet test .\test\ModernWpf.WinUI.Tests\ModernWpf.WinUI.Tests.csproj --configuration Release --framework net8.0-windows7.0 --no-build --no-restore --filter "FullyQualifiedName~WindowVisualStateTests|FullyQualifiedName~TitleBarApiTests"`

Result: 15 passed, 0 failed, 0 skipped. This covers the active window resource/template contract, captionless chrome removal, full-glass/non-client resize-edge policy, High Contrast, and title-bar behavior.

No product files were changed. Closing the true desktop-through-border transparency request as not planned; applications that explicitly need a layered transparent window can still opt out of ModernWpf chrome and own that specialized shell.
